Update debugging-workflows.md (#14997)
dotnet/docs: docs/framework/windows-workflow-foundation/debugging-workflows.md

## Debugging in the Workflow Designer

Breakpoints can be set on activities in the workflow designer by either highlighting the activity and pressing <kbd>F9</kbd> or by using the activity’s context menu. Execution of the workflow then breaks when the workflow host is run in debug mode. In the following screenshot, workflow execution is paused at a breakpoint. For more information, see [Debugging Workflows with the Workflow Designer](/visualstudio/workflow-designer/debugging-workflows-with-the-workflow-designer).

## Debugging in XAML

If a workflow has paused at a breakpoint in the designer, the workflow can also be debugged in XAML. To view the point of execution in XAML, select **XAML View** in the workflow designer when workflow execution is paused. Debugging can be switched back to the designer by re-opening the workflow in the designer from the solution explorer. For more information, see [How to: Debug XAML with the Workflow Designer](/visualstudio/workflow-designer/how-to-debug-xaml-with-the-workflow-designer).

## Debugging in Code

> Code breakpoints can be used in [!INCLUDE[netfx_current_short](../../../includes/netfx-current-short-md.md)] in the same way that they can be used in other imperative applications. Click the left margin of the code pane to create a code breakpoint, or press <kbd>F9</kbd> to place a breakpoint at the cursor location.

Reviewer: ```suggestion
To set a breakpoint, click the left margin of the code pane to create a code breakpoint, or press <kbd>F9</kbd> to place a breakpoint at the cursor location.
```

Author: @rpetrusha for being late, just noticed this now.

I think the repetition of `set a breakpoint` and `create a breakpoint` is unintentional. so I'm modifying the suggestion to:

```suggestion
To set a breakpoint, click the left margin of the code pane, or press <kbd>F9</kbd> with the cursor at the line where you want to set it.
```
